explore/src/main/scala/explore/common/ProposalQueries.scala

```diff
@@ -73,22 +61,25 @@ trait ProposalQueries:
             LargeProgramInput(
               toOActivation = toOActivation.assign,
               minPercentTime = minPercentTime.assign,
-              minPercentTotalTime = minPercentTotalTime.assign
-              // totalTime = totalTime.toInput.assign
+              minPercentTotalTime = minPercentTotalTime.orUnassign
+              // totalTime = totalTime.map(_.toInput).orUnassign
```

Comment:
Is this commented code to be addressed in a future PR?

Reply:
Yes, times is the last piece